recipes/tbb/all/conanfile.py

```diff
@@ -15,7 +15,7 @@ class TBBConan(ConanFile):
     topics = ("conan", "tbb", "threading", "parallelism", "tbbmalloc")
     settings = "os", "compiler", "build_type", "arch"
     options = {"shared": [True, False], "fPIC": [True, False], "tbbmalloc": [True, False], "tbbproxy": [True, False]}
-    default_options = {"shared": False, "fPIC": True, "tbbmalloc": False, "tbbproxy": False}
+    default_options = {"shared": True, "fPIC": True, "tbbmalloc": False, "tbbproxy": False}
```

Comment:
Agree this should be the default for this library, however, I am afraid this could somehow affect the plans regarding the introduction of https://docs.conan.io/en/latest/reference/conanfile/methods.html#self-info-shared-library-package-id

Reply:
it doesn't conflict with `shared_library_package_id`.
the only thing is recommended passage:
```
If you want to have in your dependency graph all static libraries or all shared libraries, (but not shared with embedded static ones) it can be defined with a *:shared=True option in command line or profiles, but can also be defined in recipes like:
```
so it's something optional, and I strongly believe it's better to control from the profile or command line, rather from recipe(s). if control from recipe, using wildcard `*` is too much, it should be only affected in known cases (e.g. shared `glog` needs shared `gflags`, shared `GStreamer plug-ins` need shared `GStreamer`, etc). turning all graph into shared is not an option for many reasons.